Modelica/Mechanics/MultiBody/Types.mo

```diff
@@ -117,7 +117,7 @@ are shown for a cylinder:
 </p>
 </html>"));
   type ShapeType = Modelica.Icons.TypeString
-    "Type of shape (box, sphere, cylinder, pipecylinder, cone, pipe, beam, gearwheel, spring, <external shape>)"
+    "Type of shape (box, sphere, cylinder, pipecylinder, cone, pipe, beam, gearwheel, spring, vector, <external shape>)"
```

Comment:
Can we have it between gearwheel (108) and spring (111)?

Reply:
Done.

Reply:
And now removed as part of redesign. So complete resolved.